systemtests/src/main/java/io/enmasse/systemtest/platform/apps/SystemtestsKubernetesApps.java

```diff
@@ -232,7 +236,32 @@ public static void deployAMQBroker(String namespace, String name, String user, S
             kubeCli.createNamespace(namespace);
         }
 
-        kubeCli.createSecret(namespace, getBrokerSecret(name, certBundle));
+        kubeCli.getClient().rbac().roles().inNamespace(namespace).create(new RoleBuilder()
+                .withNewMetadata()
+                .withName(name)
+                .withNamespace(namespace)
+                .endMetadata()
+                .withRules(new PolicyRuleBuilder()
+                        .addToApiGroups("")
+                        .addToResources("secrets")
+                        .addToResourceNames(name)
+                        .addToVerbs("get")
+                        .build())
+                .build());
+        kubeCli.getClient().rbac().roleBindings().inNamespace(namespace).create(new RoleBindingBuilder()
+                .withNewMetadata()
+                .withName(name)
+                .withNamespace(namespace)
+                .endMetadata()
+                .withNewRoleRef("rbac.authorization.k8s.io", "Role", name)
+                .withSubjects(new SubjectBuilder()
+                        .withKind("ServiceAccount")
+                        .withName("address-space-controller")
+                        .withNamespace("enmasse-infra")
```

Comment:
it is not always true -> you in dowsntream we do not use enmasse infra namespace

Reply:
good catch, it's fixed now, we don't need to run tests again